.devcontainer/Dockerfile

```diff
@@ -1,31 +1,52 @@
+# syntax = devthefuture/dockerfile-x
+
 ARG BASE_IMAGE=ros:humble
 
+############################################################################################################
 FROM ${BASE_IMAGE} as base
 
 USER root
 
 ENV DEBIAN_FRONTEND=noninteractive
 
+# Set the LD_PRELOAD environment variable
+ENV LD_PRELOAD=/usr/lib/libdlfaker.so:/usr/lib/libvglfaker.so
+
+# Define the workspace path
+ENV WORKSPACES=/workspaces/aoc_hri_ros2_pkg
```

Comment:
Same here, for a devcontainer, I don't know why needed.

Reply:
to remove this error:
ros@f1a7eecc10f4:~/aoc_hri_ws$ sudo ls 
ERROR: ld.so: object 'libdlfaker.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object 'libvglfaker.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.